libtmux/formats.py

```diff
@@ -7,6 +7,11 @@
 
 """
 
+from __future__ import absolute_import, unicode_literals, with_statement
+from os import environ
+
+FORMAT_SEPERATOR = environ.get("TMUX_SEPERATOR", "|")
```

Comment:
```suggestion
import os

FORMAT_SEPERATOR = os.environ.get("TMUX_SEPERATOR", "|")
```

Bikeshed: This makes it more obvious it's from `os`

Reply:
9411cc1